requirements.txt

```diff
@@ -1 +1,2 @@
+future==0.16.0
 requests==2.3.0
```

Comment:
@annakovale @bartek technically wasn't done in this PR, but we should update the `requests` requirement here. Latest is `2.12.4` I think, but we should at least use `2.10`.